index.html

```diff
@@ -365,33 +366,33 @@ <h3>Lets Connect</h3>
             <li class="languages">
               <div class="languages-title">
                 <h2>Languages</h2>
-                <img src="images/downarrow.svg" alt="" />
+                <img src="images/downarrow.svg" alt="down arrow icon" />
```

Comment:
- Optional
Your language layout is also breaking. Consider fixing it.
![image](https://user-images.githubusercontent.com/33268927/173549645-700c2e44-7a48-4e98-ab4b-c9f27e2707c9.png)